src/sync/models/centralizedOracle.js

```diff
@@ -42,12 +43,14 @@ class CentralizedOracle {
       status: 'VOTING',
       token: 'QTUM',
       name: this.name,
-      options: this.resultNames,
+      options: this.options,
       optionIdxs: Array.from(Array(this.numOfResults).keys()),
       amounts: _.fill(Array(this.numOfResults), '0'),
       resultIdx: null,
       blockNum: this.blockNum,
+      startBlock: this.bettingStartBlock,
```

Comment:
if none, set to blockNum

Reply:
done for decentralized

Reply:
so smart contract guarantee bettingStartBlock wont be null?